src/Definition/Common.php

```diff
@@ -58,6 +72,22 @@ public function isNullable()
     /**
      * @return string
      */
+    public function getDefault()
```

Comment:
`default` and `format` properties should be reflected in the `getSQLDefinition` and `toArray` methods